src/lib/viewers/image/ImageViewer.js

```diff
@@ -78,6 +80,23 @@ class ImageViewer extends ImageBaseViewer {
         super.handleAssetAndRepLoad();
     }
 
+    /**
+     * Sets mode to be AnnotationMode.NONE if the zoomed image overflows the viewport.
+     *
+     * @return {void}
+     */
+    handleZoomEvent(height, width) {
+        const viewport = {
```

Comment:
Maybe it's worth creating a helper function like `getViewportDimensions` to be shared with `zoom`?

Reply:
Yep that's a good idea.